### DISCUSSION TITLE
Is it possible to use a route like `[domain].org/api-reference` without Next.JS defaulting to API routes?

### DISCUSSION THREAD

**Discussion by polarizing** [vercel/next.js#15648] [Help]
It seems like Next.JS will not export any page with the keyword `api` in the file name. It seems like Next has reserved that keyword for API routes which do not export statically.

How would I go about building a static page with a keyword like `api` in the route?

For example: [domain].org/docs/api-reference works fine but [domain].org/api-reference will not build.

**Comment by timneutkens** [comment]
Are you on the latest version of Next.js? If not please upgrade. If you are please create an issue with a complete clear and concise reproduction (e.g. a github repository).
